docs/bookstore/page-0/page-0-intro.md

```diff
@@ -0,0 +1,149 @@
+
+6. [**Advanced event filtering**](../page-6/pg6-review-svc-2.md){:target="_blank"}: Integrate all components by receiving "analyzed reviews" via Broker (using a trigger) and storing them into the database. This includes learning about Triggers and Filters.
+
+7. [**Connect with External Services/API**](../page-7/pg7-slack-camel.md){:target="_blank"}: Learn how to connect your application with external services and APIs using Knative Eventing and Apache Camel-K integrations.
+
+## **How to properly learn?**
+
+### Preview the Final Bookstore: 1-Minute Demo Video
+
+To help you visualize what you'll be creating, we've prepared a brief demo video. This two-minute preview showcases the final bookstore application, providing a clearer understanding of what you can expect to build.
+
+<iframe width="1924" height="500" src="https://www.youtube.com/embed/5D8pTcQSacw" title="Knative E2E Bookstore Tutorial  - 1-min Demo" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+???+ bug "Video not working above?"
+    If the video is not working, you can watch it [here](https://www.youtube.com/watch?v=5D8pTcQSacw&ab_channel=Knative){:target="_blank"}.
+
+### Step-by-Step Guidance for Beginners
+
+![Beginner Guide](images/9.png)
+
+This tutorial is meticulously structured to be beginner-friendly, featuring detailed, step-by-step instructions for building the sample app. Simply follow the sequence we've laid out: each section builds upon the previous, guiding you through the construction of the application. Should you encounter any hurdles, the Knative community is a fantastic resource for support. Don't hesitate to ask questions and seek advice. Check the [Help](#help) section below!
+
+### Accelerated Learning Path for Advanced Users
+
+![Advanced Users Guide](images/10.png)
+
+If you find the tutorial too basic, or if it covers familiar territory, feel free to adjust your learning approach. Each section of the tutorial is accompanied by concise graphics that summarize key concepts. Advanced learners can choose to focus on these graphics to grasp the essentials faster, streamlining the learning experience without sacrificing depth or understanding.
+
+## **Help**
+
+![Help Image](images/11.png)
+
+Join the supportive Knative community via the Cloud Native Computing Foundation (CNCF) Slack, particularly the [#knative](https://cloud-native.slack.com/archives/C04LGHDR9K7){:target="_blank"} channel. Before posting your questions, please search to see if they've already been answered. Your feedback on this tutorial is invaluable, so don't hesitate to reach out with suggestions or questions.
+
+## **Next Step**
+
+![Next Step Image](images/12.png)
+
+Clone the repo first
+```sh
+git clone https://github.com/knative/docs.git
+```
```

Comment:
This `clone` seems a little hidden down here and perhaps better to add it to the next `Environment Setup` page?